.github/workflows/audits.yml

```diff
@@ -0,0 +1,23 @@
+name: Run Rust audits
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Looks like this runs nightly. What happens if an audit fails? Does a github issue get opened up?

Reply:
From https://github.com/fermyon/spin/pull/1023#discussion_r1072606931

> The main reason it's not on PRs is that we should only trust an audit coming from a maintainer, not from someone submitting a PR.

> Ideally, when maintainers would submit a PR, any changes in dependencies would be included in the PR. But that should not be the case with non-maintainers — hence running this every day, and maintainers should ensure the check is passing.